modules/apps/headless/headless-cms/headless-cms-impl/src/main/java/com/liferay/headless/cms/internal/resource/v1_0/BulkActionResourceImpl.java

```diff
@@ -18,4 +57,289 @@
 	scope = ServiceScope.PROTOTYPE, service = BulkActionResource.class
 )
 public class BulkActionResourceImpl extends BaseBulkActionResourceImpl {
+
+	@Override
+	public EntityModel getEntityModel(MultivaluedMap multivaluedMap)
+		throws Exception {
+
+		return _entityModel;
+	}
+
+	@Override
+	public BulkActionTask postBulkAction(
+			String search, Filter filter, BulkAction bulkAction)
+		throws Exception {
+
+		if (BulkAction.Type.DELETE_BULK_ACTION.equals(bulkAction.getType())) {
+			return _executeDeleteBulkAction(
+				bulkAction,
+				_getBulkActionItemMap(
+					bulkAction.getBulkActionItems(), filter, search,
+					GetterUtil.getBoolean(bulkAction.getSelectAll())));
+		}
+
+		throw new UnsupportedOperationException();
+	}
+
+	private BulkActionTask _addBulkActionTask(String type) throws Exception {
+		ObjectEntry bulkActionTask = _objectEntryLocalService.addObjectEntry(
+			0, contextUser.getUserId(), _getBulkActionTaskObjectDefinitionId(),
+			ObjectEntryFolderConstants.PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+			null,
+			HashMapBuilder.<String, Serializable>put(
+				"actionName", type
+			).put(
+				"executionStatus", "STARTED"
+			).put(
+				"type", type
+			).build(),
+			new ServiceContext());
+
+		return new BulkActionTask() {
+			{
+				Map<String, Serializable> values = bulkActionTask.getValues();
+
+				setActionName(
+					() -> GetterUtil.getString(values.get("actionName")));
+
+				setAuthor(bulkActionTask::getUserName);
+				setCreatedDate(bulkActionTask::getCreateDate);
+				setExecuteStatus(
+					() -> GetterUtil.getString(values.get("executionStatus")));
+				setExternalReferenceCode(
+					bulkActionTask::getExternalReferenceCode);
+				setId(bulkActionTask::getObjectEntryId);
+				setType(() -> GetterUtil.getString(values.get("type")));
+			}
+		};
+	}
+
+	private void _addBulkActionTaskItem(
```

Comment:
is really needed this method? I mean.. it's a "single line" method called only once... at this point is the same to call directly the addObjectEntry